src/main/java/com/akathist/maven/plugins/launch4j/FileUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.akathist.maven.plugins.launch4j;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+public class FileUtils {
```

Comment:
`final`?

Reply:
If you want to. It's a bag for copy-paste funcs anyway.